scripts/nullaway-suppression-remover/src/suppression_remover/core.py

```
@@ -0,0 +1,847 @@
+    Run a build for the target module.
+
+    *build_cmd* overrides the default Gradle invocation.  The default
+    compiles every source set that contains Java files::
+
+        ./gradlew :<module>:compileJava :<module>:compileTestJava ... --continue
+
+    Returns ``(combined_output, success)``.
+    """
+    cmd = build_cmd or default_gradle_cmd(module_dir, project_root)
+
+    result = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)